src/test/java/test/integration/algeron/AddArquillianAlgeronRetrieverTest.java

```diff
@@ -23,15 +20,8 @@ public void should_register_folder_retriever() throws TimeoutException {
             .execute("arquillian-algeron-setup-provider --contracts-library pact")
             .execute("arquillian-algeron-setup-retriever --retriever folder --contract-folder /tmp");
 
-        final FileResource<?> arquillianXml = extractTestResource(project, "arquillian.xml");
-        assertThat(arquillianXml.exists()).isTrue();
-
-        final ArquillianConfig arquillianConfig = new ArquillianConfig(arquillianXml.getResourceInputStream());
-        assertThat(arquillianConfig.isExtensionRegistered("algeron-provider")).isTrue();
-        final String configuration = arquillianConfig.getContentOfNode("extension@qualifier=algeron-provider/property@name=retrieverConfiguration");
-        assertThat(configuration).isNotNull();
-        assertThat(configuration)
-            .contains("provider: folder", "contractsFolder: /tmp");
+        assertThat(project).hasArquillianConfig().withExtension("algeron-provider")
+            .withProperty("retrieverConfiguration", "provider: folder\ncontractsFolder: /tmp");
```

Comment:
I think previous way was more readable. What if we add `\n\n` which should be conceptually irrelevant? We should match strings as such, but key-values, right? Can you think about making it even more pleasant to use?